.github/workflows/build_and_test.yaml

```diff
@@ -44,7 +44,7 @@ jobs:
           version: v1.46.2
           args: --build-tags=e2e
       - name: Build
-        run: make build-all
+        run: sudo make build
```

Comment:
curious why is `sudo` needed ?

Reply:
This is something weird for me, if I do not use sudo, an error with no permission will show up.

Reply:
You shouldn't need that in CI still.

Reply:
I do not know why, I did not meet this issue in other repo, I will have a try to remove this.